### PR TITLE
jsdialog: added button in about dialog to open WidgetTestDialog

### PR DESCRIPTION
**Pull request by Rash419**
Signed-off-by: Rash419 <rashesh.padia@collabora.com>
Change-Id: Ib525515f27640114f5d8bc83314b63f006893d37

* Target version: master

### CODE REVIEW

**Comment by Rash419** [comment]
core patch : https://gerrit.libreoffice.org/c/core/+/132891

File: browser/html/cool.html.m4 [hunk 1]
```diff
@@ -283,6 +283,7 @@ m4_ifelse(MOBILEAPP,[true],
             <div style="margin-inline-end: auto;"><div id="lokit-version" dir="ltr"></div></div>
             m4_ifelse(MOBILEAPP,[],[<div id="served-by"><span id="served-by-label"></span>&nbsp;<span id="os-info"></span>&nbsp;<wbr><span id="coolwsd-id"></span></div>],[<p></p>])
             <div id="slow-proxy"></div>
+            <div id="js-dialog">JSDialogs: <a href="javascript:void(app.socket.sendMessage('uno .uno:WidgetTestDialog'))">View widgets</a></div>
```
Comment (eszkadev) [inline browser/html/cool.html.m4:286]: I think we want it only in DEBUG build, could you check if there is some macro for it?
Like above for mobile app: `m4_ifelse(MOBILEAPP`

**Review by eszkadev — APPROVED** [review]

**Comment by eszkadev** [comment]
It would be nice if that link would close the "About dialog", can we access vex from window object?  vex has some method like closeAll if I remember...

**Review by eszkadev — APPROVED** [review]

**Comment by eszkadev** [comment]
before push I'm waiting for core patch for co-22.05

**Comment by eszkadev** [comment]
"iOS against Core co-22.05" passed before and now is broken also in other PRs

**Pull request merged**